README.md

```diff
@@ -19,9 +19,86 @@ Assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running the demo on Windows
+
+Prerequisites:
+
+- [Docker desktop](https://docs.docker.com/docker-for-windows/install/)
+
+### Step 1
+
+First test that the docker installation is working. Open a Powershell prompt (press Windows+S and type Powershell) and run:
+
+```shell
+docker run hello-world
+```
+
+This should show a message that your Docker installation is working correctly. If so, we can proceed to the installation of evidence tool, otherwise we suggest to check the [Docker troubleshooting page](https://docs.docker.com/docker-for-windows/troubleshoot/).
+
+### Step 2
+
+[Download](https://github.com/ADAH-EviDENce/evidence/archive/master.zip) a copy of evidence archive and extract its contents on your machine.
+
+### Step 3
+
+Open a Powershell prompt:
+Change your current working directory to where you extracted the files. For instance:
+
+```shell
+cd C:\Users\JohnDoe\Downloads\evidence-master\evidence-master
+```
+
+The demo can be started with the below command. Keep this Powershell window open and running during the demo.
+
+```shell
+$Env:EXPERIMENT="demo"
+docker-compose up --build
+```
+
+The command above downloads necessary Docker images, builds all the Docker images and starts the demo.
+
+The command prints many log messages. If all goes well, the last lines of the output should be:
+
+```shell
+...
+indexer_1        | Indexing done.
+evidence-master_indexer_1 exited with code 0
+```
+
+In some cases, the docker containers cannot access the necessary folder. In this case, you get output like the following:
```

Comment:
Can you explain what are the necessary folders? 
Also what do you mean with ``some cases``?

Reply:
My sggestion is to turn this argument around: "_if you get output like the following_"
```powershell
powershell stuff
```
the docker containers cannot access the necessary folder C:\Windows\makes\everything\more\difficult.

Reply:
Its windows 10 pro vs home, clearified in readme + clearified folder contents